applications/sport-store/index.html

```diff
@@ -0,0 +1,125 @@
+                   <li class="quick_links_item">
+                        <a class="qlink" href="#">Register for SportStore Giftcard</a>
+                   </li>
+                </ul>
+            </div>
+            <div class="header_signin">
+                      <img class="user_icon" alt="User Icon" src="/applications/sport-store/images/user.png"/> 
+                      <a class="signin" href="#">Sign In/Create Account</a>
+            </div>
+        </div>
+
+        <div id="logo-header" class="page_margin_container">
+            <div class="page_header_logo">
+                <a href="#">
+                     <img class="logo_image" alt="Sport Store Logo" src="/applications/sport-store/images/logo.png" />
+                </a>
+            </div>
+
+            <div class="header_search">
+              <form class="header_search_form" action="">
+                <input class="search_input" type="text" size="60" placeholder="Search Sport Item Name ...." name="search">
+                <button class="search_button" type="submit">
+                     <img src="/applications/sport-store/images/searchicon.png"/>  
+                </button>
+              </form>
+            </div> 
+
+            <div class="header_cart">
+                 <a href="#" class="header_cart_trigger">
+                     <span class="header_cart_count">0</span>
+                 </a>
+            </div>
+        </div>
+      </header>
+
+      <main id="page-content">
+       <div class="page_margin_container"> 
+            <div class="page_content_aside">
+                <aside class="aside_menu">
+                    <ul class="ul_vertical_menu">
+                        <li class="li_vertical_menu"><a class="active" href="#">All</a></li>
+                        <li class="li_vertical_menu"><a href="#">Watersports</a></li>
+                        <li class="li_vertical_menu"><a href="#">Soccer</a></li>
+                        <li class="li_vertical_menu"><a href="#">Chess</a></li>
+                    </ul>
+                </aside>
+                <div class="aside_news">
+                    <h3>Host news</h3>
+                    <p>
+                      <strong>COVID-19 in Sask</strong> 17 cases in Saskatoon area now
+                      connected to Brandt Industries.<br> 
+                      <a target="_blank" href="https://www.cbc.ca/news/canada/saskatchewan/covid-19-saskatchewan-september-21-1.5732916">See more detail ... </a> 
+                    </p>
+                </div>
+            </div>
+  
+  
+        <div class="page_content_center">
+          <section>
+            <h2>Navigation Bar</h2>
+          </section>
+          <section>
+            <table>
+              <tr>
+                <th>Sport Items</th>
+                <th>Price</th>
+              </tr>
+              <tr>
+                <td>
+                  <head>
+                    Men's Terrex Agravic Flow Shoes
+                  </head>
```

Comment:
You shouldn't be using a `<head>` element here.

Reply:
Oh ! I didn't know why <head> element is occurred in this code. Thanks !